src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -244,10 +253,14 @@ public void Should_Set_Commandline_Switches()
                         "--stoponerror \"--work=/Working/out\" \"--out=/Working/stdout.txt\" " +
                         "\"--err=/Working/stderr.txt\" --full " +
                         "\"--result=/Working/NewTestResult.xml;format=nunit2;transform=/Working/nunit2.xslt\" " +
```

Comment:
@ChrisMaddock What will happen when combining `format` and `transform`? Does it give an error or does one have precedence over the other?